<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgNotification</class>
 <widget class="QDialog" name="DlgNotification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>137</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>132</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>160</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lbl_icon">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>64</width>
     <height>64</height>
    </rect>
   </property>
   <property name="text">
    <string>Icon</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_autohide">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Autohide</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chk_ignore">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>90</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Don't show again</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>90</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_action">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>90</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Update Resource Host</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="lbl_message">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>491</width>
     <height>70</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::TextSelectableByMouse</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
